scripts/build-windows.sh

```diff
@@ -15,7 +15,6 @@ mkdir ./build/windows/cardano-hw-cli
 
 # Build executable
 yarn nexe ./dist/index.js -o ./build/windows/cardano-hw-cli/cardano-hw-cli -t windows-x64-12.16.2
-mv ./build/windows/cardano-hw-cli/cardano-hw-cli.exe ./build/windows/cardano-hw-cli/cardano-hw-cli 
```

Comment:
why this line is being removed?

Reply:
This line removes renaming of `cardano-hw-cli.exe` to `cardano-hw-cli`. Related issue: https://github.com/vacuumlabs/cardano-hw-cli/issues/44